src/config/NetworkConfig.ts

```diff
@@ -72,6 +73,7 @@ const defaultNetworkConfig: NetworkConfig = {
         harvestingMosaicId: '5F160D7851F3CB30',
         defaultDynamicFeeMultiplier: 1000,
         epochAdjustment: 1573430400,
+        totalChainImportance: 1,
```

Comment:
you can use the default value in the testnet, which will be the same as main net after launch

http://api-01.us-west-1.testnet.symboldev.network:3000/network/properties